src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.Unix.cs

```diff
@@ -226,61 +226,5 @@ private void SendFileInternal(string? fileName, ReadOnlySpan<byte> preBuffer, Re
                 Send(postBuffer);
             }
         }
-
-        private async Task SendFileInternalAsync(FileStream? fileStream, byte[]? preBuffer, byte[]? postBuffer)
```

Comment:
The Unix-implementation is also based on SendPackets*, so Windows and Unix got unified here.
But I assume that now on Unix there are more allocations happening than before.

We can fix that once we base it on SAEA based version.

\* from https://github.com/dotnet/runtime/issues/42591#issuecomment-829391326

> If we base it on SendPacketsAsync, this would work for both Windows and Unix, which is preferred -- we don't want to have more platform-specific code here if we can avoid it.